java/README.md

```diff
@@ -90,20 +58,130 @@ Other useful gradle developer commands:
 * `./gradlew :examples:run` to run client examples (make sure you have a running redis on port `6379`)
```

Comment:
```suggestion
* `./gradlew :examples:run` to run client examples (make sure you have a running valkey on port `6379`)
```

Reply:
i don't think there is a `valkey` port yet, that's why it still has to be redis.

Reply:
I think you can mention both or valkey only

Reply:
https://stackoverflow.com/questions/78494458/valkey-cache-in-c-sharp this seems to suggest the default port for Valkey is the same as for redis (unsurprisingly) so it's probably okay to commit this suggestion.